package.json

```diff
@@ -81,14 +80,15 @@
     "karma-coverage-istanbul-reporter": "^2.0.4",
     "karma-jasmine": "^1.1.1",
     "karma-jasmine-html-reporter": "^1.4.0",
-    "ng-packagr": "^4.4.0",
+    "ng-packagr": "^5.1.0",
     "npm-install-peers": "^1.2.1",
     "protractor": "^5.3.1",
     "ts-node": "^7.0.1",
-    "tsickle": "^0.33.1",
-    "tslint": "^5.9.1",
-    "typescript": "~3.1.3",
-    "zone.js": "^0.8.26"
+    "tsickle": "^0.35.0",
+    "tslib": "^1.10.0",
+    "tslint": "^5.18.0",
+    "typescript": "~3.4.5",
```

Comment:
I think this version won't work with Angular 7, in case we try to run the project in development mode for testing with that Angular version.